middleware/requestid/custom_generator/main.go

```diff
@@ -31,7 +31,7 @@ func main() {
 
 	h.Use(
 		// define your own request id generator here
-		requestid.New(requestid.WithGenerator(func() string {
+		requestid.New(requestid.WithGenerator(func(ctx context.Context, c *app.RequestContext) string {
```

Comment:
Is this a break change in `requestid`?

Reply:
Yes, requestid released an intermediate version for this feature